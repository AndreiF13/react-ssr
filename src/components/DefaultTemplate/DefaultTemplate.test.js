const DefaultTemplate = require('./DefaultTemplate')

test('DefaultTemplate renders without exploding', () => {
  const wrapper = shallow(<DefaultTemplate />)
  expect(wrapper.length).toEqual(1)
  expect(snapshot(wrapper)).toMatchSnapshot()
})
