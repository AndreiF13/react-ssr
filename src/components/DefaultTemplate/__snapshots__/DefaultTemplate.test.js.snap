// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultTemplate renders without exploding 1`] = `
<html
  lang="en-gb"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="IE=edge"
      httpEquiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      content="yes"
      name="mobile-web-app-capable"
    />
    <link
      href="/assets/styles.css"
      rel="stylesheet"
    />
    <link
      href="/favicon.ico"
      rel="icon"
      type="image/x-icon"
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      }
    />
  </head>
  <body>
    <div
      id="root"
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__STATE = undefined;",
        }
      }
    />
    <script
      src="/assets/bundle.js"
    />
  </body>
</html>
`;
